email/email.go

```diff
@@ -0,0 +1,25 @@
+package email
```

Comment:
not a good package name, this package is depends on mailgun exclusevly, so it;s better to call it mailgun